profiles/package.mask

```diff
@@ -832,31 +832,6 @@ dev-python/shiboken6
 dev-python/pyside6
 dev-python/pyside6-tools
 
-# Jimi Huotari <chiitoo@gentoo.org> (2022-08-02)
```

Comment:
Let's include the pyside6 mask above as well.

Reply:
And I suppose the games-emulation/pcsx2 mask too..

Reply:
Yeah please do, it was only separate because wanted a custom mask message for surprised users.

Reply:
Also, what about USE=qt6? Not many packages will be able to use qt6 if not unmasked on arch/amd64/use.mask (I'd leave the base mask on for now).

Flag is typically off by default for now so it shouldn't have a big impact unless people opt-in.

Reply:
Definitely forgot about the flag, yeah, and did not think of the other packages.

Thanks!

Reply:
> I'd leave the base mask on for now

@ionenwks not a fan of this... we're really late on qt6 and it's about time we give it more testing. I can do my part on ppc64le.

Reply:
Well, once it's unmasked we can go through normal keywording requests and /then/ unmask it on ppc64le and such.